modules/culling/src/lib/oriented-bounding-box.js

```diff
@@ -35,6 +35,22 @@ export default class OrientedBoundingBox {
     this.halfAxes = new Matrix3(halfAxes);
   }
 
+  // Create OrientedBoundingBox from OBB based on quaternion
+  static fromQuaternionObb(quaternionObb) {
```

Comment:
More readable if you type out the parameters:
```suggestion
  fromQuaternionObb({center; halfSize, quaternion}) {
```